onnxmltools/convert/coreml/convert.py

```diff
@@ -73,4 +73,7 @@ def convert(model, name=None, initial_types=None, doc_string='', targeted_onnx=o
     if len(metadata_props) > 0:
         onnx_model.metadata_props.extend(metadata_props)
 
+    # Check if model is valid
```

Comment:
This does not seem to belong here. The converter should never produce something that makes checker fail.

Reply:
You're right. If we call the checker here, similar things can be prevented in the future and our code could become more robust. Should I remove it?

Reply:
yes please